features/schema.py

```diff
@@ -0,0 +1,34 @@
+import graphene
+import graphql_geojson
+from django.apps import apps
+from graphene import relay
+from graphene_django import DjangoConnectionField, DjangoObjectType
+
+from features import models
+
+
+class SourceType(DjangoObjectType):
+    class Meta:
+        model = models.SourceType
+        interfaces = (relay.Node,)
+
+
+class FeatureTranslations(DjangoObjectType):
+    class Meta:
+        model = apps.get_model("features", "FeatureTranslation")
+
+
+class Feature(graphql_geojson.GeoJSONType):
+    class Meta:
+        model = models.Feature
+        geojson_field = "geometry"
+        interfaces = (relay.Node,)
+
+    name = graphene.String()
+
+
+class Query(graphene.ObjectType):
```

Comment:
Currently the schema has `name` and `translations` separately in `FeatureProperties`. Is it possible to combine these to match the idea of `LocalizedString` or is that still in design phase? Also, could the resolver be made to support a language parameter?

This comment is not blocking, just curious about the future implementation (and whether tickets about these questions should be made).